File: python/Arrays/day7/mcones.py
n = int(input("Enter the size of array: "))

arr = list(map(int, input(f"Enter {n} numbers: ").split()))

cnt = 0
maxi = 0

for i in range(n - 1):
    if arr[i] == 1:
        cnt += 1
        maxi = max(maxi, cnt)
    else:
        cnt = 0

print("Consecutive Ones is: ", maxi)

File: java/Arrays/Day7/MCONES.java
package Day7;
import java.util.*;

public class MCONES {
    public static void main(String[] ones) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter the size of Array: ");
        int n = sc.nextInt();


        int arr[] = new int[n];
        System.out.print("Enter " + n + " " + "numbers: ");
        for (int i = 0; i < n; i++) {
            arr[i] = sc.nextInt();
        }

        int cnt = 0;
        int maxi = 0;

        for (int i = 0; i < n; i++) {
            if (arr[i] == 1) {
                cnt++;
                maxi = Math.max(maxi, cnt);
            } else {
                cnt = 0;
            }
        }

        System.out.print("Consecutive Ones is: " + maxi);

        sc.close();
    }
}